src/main/java/net/sf/jabref/openoffice/OOBibStyle.java

```diff
@@ -705,15 +705,10 @@ private String getCitationMarkerField(BibEntry entry, BibDatabase database, Stri
             String content = BibDatabase.getResolvedField(s, entry, database);
 
             if ((content != null) && !content.trim().isEmpty()) {
-                if (fieldFormatter != null) {
-
-                    if (field.equals(authorField) && StringUtil.isInCurlyBrackets(content)) {
-                        return "{" + fieldFormatter.format(content) + "}";
-                    }
-                    return fieldFormatter.format(content);
+                if (field.equals(authorField) && StringUtil.isInCurlyBrackets(content)) {
+                    return "{" + fieldFormatter.format(content) + "}";
```

Comment:
Yes, the fieldFormatter is initialized on construction. I think it is a
left-over from the plugin time.

(Of course, the initialization can fail, but then one should null check
after every new-call...)